runtime/autoload/man.vim

```diff
@@ -142,8 +131,8 @@ endfunction
 
 function s:find_page(sect, page) abort
   let where = system('/usr/bin/man '.s:man_find_arg.' '.s:cmd(a:sect, a:page))
-  if where !~ "^/"
-    if matchstr(where, " [^ ]*$") !~ "^ /"
+  if where !~# "^/"
+    if matchstr(where, " [^ ]*$") !~# "^ /"
```

Comment:
This is some crazy code: three (!) regexes used and two levels of `if`, while one regex can be used and zero levels:

```VimL
return (where =~# '^/\| /')
```